### DISCUSSION TITLE
I need a flash dump file for z110 hsgq

### DISCUSSION THREAD

**Discussion by DonuyThompson** [Anime4000/RTL960x#340] [General]
![2024-11-29_110516](https://github.com/user-attachments/assets/272eb3ee-c50a-4b52-83aa-8fe00dd50eb0)
Hello, during the update process the power went out, the device does not work, I am looking for flash dump

z110 hsgq
![2024-11-29_110457](https://github.com/user-attachments/assets/1ccbfadb-740b-4ffd-831a-c4a8c46c85f9)

**Comment by Anime4000** [comment]
[ZQ-110.zip](https://github.com/user-attachments/files/17960017/ZQ-110.zip)
here

**Reply by DonuyThompson** [reply]
Thanks 
![2024-11-30_193653](https://github.com/user-attachments/assets/08bb63ff-2478-47ba-820e-820d6fa8cffc)
but it didn't work
This is my device

**Reply by Anime4000** [reply]
well, I didn't want to pull the SPI out, my soldering skill are bad, haha

**Reply by DonuyThompson** [reply]
Even my skills are bad, but I do it, please help me, it's my friend's device and now I'm in trouble with it
